packages/workspace/src/node/default-workspace-server.ts

```diff
@@ -108,6 +108,17 @@ export class DefaultWorkspaceServer implements WorkspaceServer {
         });
     }
 
+    async removeRecentWorkspace(uri: string): Promise<void> {
+        // Get the list of recent workspaces.
+        const recent = await this.getRecentWorkspaces();
+        // Filter out the given workspace by uri.
+        const updated: string[] = recent.filter(e => e !== uri && e.length > 0);
```

Comment:
We should probably use a better name than `updated`.
A standard loop is likely sufficient, there is no need to iterate over the whole array when a match is found since the `uris` are unique.

Reply:
Done, updated as per our offline discussion. so the `updated` variable has been removed and the loop remains as is.